src/lib/Components/Inbox/Conversations/ZeroStateInbox.tsx

```diff
@@ -52,22 +51,22 @@ const Icon = styled(Image)`
 export default () =>
   <CenteredView>
     <VerticalLayout>
-      <Title>Buying Art on Artsy is Simple </Title>
+      <Title>BUYING ART ON ARTSY IS SIMPLE</Title>
       <HorizontalLayout>
         <Icon source={require("../../../../../images/find.png")} />
-        <SmallHeadline>Follow artists and find works you love</SmallHeadline>
+        <SmallHeadline>Follow artists & categories and find works you love.</SmallHeadline>
```

Comment:
This mixture of ampersand and “and” is the one thing I like less. Can this be a list and use of the Oxford comma?

Reply:
Think `Follow artists and categories to find works you love` says the same thing, and is simpler - /cc @katarinabatina  ( I took the words from your white BG sketch doc )

Reply:
Good call 👍